docs/BEMAN_STANDARD.md

```diff
@@ -577,6 +577,61 @@ add_subdirectory(beman) # Don't do this
 add_subdirectory(optional) # Don't do this
 ```
 
+###  **[CMAKE.IMPLICIT_DEFAULTS]**
+
+**RECOMMENDATION**: Where CMake commands have reasonable default values, and the project does not intend to change those values, the parameters should be left implicitly defaulted rather than enumerated in the command.
+
+Example:
+
+```CMake
+# Not recommended
+# Explicitly enumerated defaults for install artifact destinations
+install(
+    TARGETS beman.project
```

Comment:
```suggestion
    TARGETS beman.exemplar
```
maybe using exemplar instead?

Reply:
I didn’t say that. Is this exemplar for a library?  If yes, exemplar library should be used

Reply:
The other examples int his sections use `smart_pointer`, `exemplar`, and `optional` mostly a random. While I agree this should be standardized to _something_, I don't think it should be `exemplar` and I don't think it should be in this PR.

Reply:
any section at least use the exemplar example right? which you may also inspect when using cookiecutter

Reply:
can I sugget that we do something like beman.<project_name> ? We probably shouldn't have examples from 3 different things without some text that says something along the lines of: "For example in the project called 'optional' this would be 'beman.optional'

Reply:
or after looking again perhaps something along these lines...

In the cmake section the text `project` will be replaced by the name of the project.  So for the `utf_view` project the text `beman.project` becomes `beman.utf_view`